cartographer_ros/configuration_files/backpack_3d_localization.lua

```diff
@@ -14,7 +14,9 @@
 
 include "backpack_3d.lua"
 
-TRAJECTORY_BUILDER.pure_localization = true
+TRAJECTORY_BUILDER.pure_localization_trimmer = {
```

Comment:
I am not too familiar with ROS but I am not sure how the localization trimmer is related to the MarkerArray